src/app/scenarios/create-scenario/createScenario.component.js

```diff
@@ -0,0 +1,14 @@
+import {RouteParams, Router} from 'angular2/router';
```

Comment:
follow dash-n-dot naming convention
this file should've been named `screate-scenario.component.js`
see https://angular.io/styleguide#!#naming for details